src/basic-kokkos/REDUCE3_INT-Kokkos.cpp

```diff
@@ -0,0 +1,151 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "REDUCE3_INT.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#include <limits>
+#include <iostream>
+
+namespace rajaperf 
+{
+namespace basic
+{
+
+
+void REDUCE3_INT::runKokkosVariant(VariantID vid)
+{
+  const Index_type run_reps = getRunReps();
+  const Index_type ibegin = 0;
+  const Index_type iend = getActualProblemSize();
+
+  REDUCE3_INT_DATA_SETUP;
+
+  //Declare KokkosView that will wrap the pointer to a vector
+
+  auto vec_view = getViewFromPointer(vec, iend); 
+
+#if defined(RUN_KOKKOS)
+
+  switch ( vid ) {
+
+    case Base_Seq : {
```

Comment:
remove the Base_seq and Lambda_Seq variants. They are repeats of what's in the basic/REDUCE3_INT-Seq.cpp file and don't depend on RAJA or Kokkos.